interface/client/mistAPIBackend.js

```diff
@@ -17,15 +19,7 @@ var filterId = function(str) {
 };
 
 
-var sound = {
-    bip: document.createElement('audio'),
-    bloop: document.createElement('audio'),
-    invite: document.createElement('audio'),
-};
-sound.bip.src = 'file://'+ dirname + '/../../sounds/bip.mp3';
```

Comment:
This was throwing errors in the JS console on Mist startup, because `dirname` is not available in the Meteor client-side. Neither is `__dirname` or any of the standard Node.js stuff, though Meteor 1.3 probably has support for it. In any case, since the file is being loaded from the local FS delays should be minimal.